articles/libraries/lock/v11/migration-legacy-flows.md

```diff
@@ -0,0 +1,171 @@
+
+## User Profiles 
+
+When using the legacy authentication flows, the entire user profile is returned in ID tokens and from `/userinfo`. For example:
+
+```json
+{
+    "email": "demo@user.com",
+    "picture": "https://s.gravatar.com/avatar/1a9b4fabea79ed763b435793887fb67b?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Fde.png",
+    "nickname": "demo",
+    "name": "demo@user.com",
+    "user_metadata": {
+        "favorite_color": "blue"
+    },
+    "app_metadata": {
+        "roleName": "foobar"
+    },
+    "roleName": "foobar",
+    "email_verified": false,
+    "user_id": "auth0|59df6364209a29662e321756",
+    "clientID": "tkQfyt2o4uy5RJwhJ3g651BSY4Fwegex",
+    "identities": [
+        {
+            "user_id": "59df6364209a29662e321756",
+            "provider": "auth0",
+            "connection": "Local-Users",
+            "isSocial": false
+        }
+    ],
+    "updated_at": "2017-12-13T14:43:01.575Z",
+    "created_at": "2017-10-12T12:43:16.682Z",
+    "sub": "auth0|59df6364209a29662e321756"
+}
+```
+
+The new user profile conforms to the OIDC specification, which allows for certain [standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) to be available in the response:
+
+```json
+{
+    "sub": "auth0|59df6364209a29662e321756",
+    "nickname": "demo",
+    "name": "demo@user.com",
+    "picture": "https://s.gravatar.com/avatar/1a9b4fabea79ed763b435793887fb67b?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Fde.png",
+    "updated_at": "2017-12-13T14:44:51.788Z"
+}
+```
+
+The contents will vary depending on which [scopes](/scopes) are requested. You will need to adjust the scopes you request when configuring Auth0.js or Lock so all the claims you need are available in your application. Note that you can add custom claims to return whatever data you want (e.g. user metadata), as described in [this example](/scopes/current#example-add-custom-claims).
+
+Another approach to get the full user profile is to use the [Management API](/api/management/v2) instead of getting it through the authentication flow, as described in the next section.
+
+## User Profile with Management API
+
+In the legacy flows, the [Management API](/api/management/v2) supported authentication with an `id_token`. This approach has been deprecated, and now you need to call it with an `access_token`. 
+
+To get an get an `access_token`, you need to ask Auth0 for one using the `https://'${account.namespace}/api/v2/` audience. Auth0 does not currently support specifying two audiences when authenticating, so you will need to still use your application's API audience when initializing Lock or auth0.js. Once the user is authenticated, you can use `checkSession` to retrieve a Management API `access_token`, and then call the [getUser() endpoint](/api/management/v2#!/Users/get_users_by_id).
```

Comment:
`https://'${account.namespace}/api/v2/`

^ is there an extra single quote ' in there?

Reply:
Yes there was (fixed), good catch 👍